triaging.rst

```diff
@@ -47,6 +47,35 @@ the Python Software Foundation, always follow the `PSF Code of Conduct`_.
 
    Existing members with the "Developer" role on the `issue tracker`_ are welcome to
    self-nominate for triage team membership if they are interested.
+   
+GitHub Labels
+''''''''''''''''
+
+An important component of triaging PRs and issues involves appropriately 
+categorizing them through the usage of labels. Here are several of the
+common ones utilized by the Python triage team:
+
++-----------------------+----------------------------------------------------------------------------+
+| Label                 | Description                                                                |
++-----------------------+----------------------------------------------------------------------------+
+| needs backport to X.Y | Used for PRs which are appropriate to backport to branches prior to        |
+|                       | master. Generally, backports to the maintenance branches are primarily     |
+|                       | bugfixes and documentation clarifications. Backports to the security       |
+|                       | branches are strictly reserved for PRs involving security fixes, such as   |
+|                       | crashes, privilege escalation, and DoS.                                    |
++-----------------------+----------------------------------------------------------------------------+
+| skip issue            | Used for PRs which involve trivial changes, such as typo fixes,            |
+|                       | comment changes, and section rephrases. The majority of PRs require        |
+|                       | an issue to be attached to, but if there are no code changes and the       |
+|                       | section being modified retains the same meaning, this label might be       |
+|                       | appropriate. It is also used commonly for backports.                       |
```

Comment:
I think the "It is also used commonly for backports" can be removed. More accurately, this label got automatically re-applied from the master PR to the backport PR.